src/frontend/js/utils/test/render.tsx

```diff
@@ -65,7 +65,11 @@ export const render: RenderFunction = (
       rerenderOptions?: Partial<Omit<AppWrapperProps, 'testingLibraryOptions'>>,
     ) => {
       return renderResult.rerender(
-        Wrapper === null ? element : <Wrapper {...rerenderOptions}>{rerenderElement}</Wrapper>,
+        Wrapper === null ? (
+          rerenderElement
+        ) : (
+          <Wrapper {...rerenderOptions}>{rerenderElement}</Wrapper>
+        ),
```

Comment:
But as the first render renders the element with Wrapper, the rerender should do the same thing nope?

Reply:
it's already the case, if it was render with wrapper the first time, it'll be rerender with wrapper.

Reply:
I just don't understand why you need to wrap again the element in the wrapper 🤔

Reply:
it's how rerender works. This way it's allow us to update props on rerender.

```ts
(property) rerender: (ui: React.ReactNode) => void
```